test/util/validators_test.py

```diff
@@ -84,7 +85,7 @@ def test_comparable_with_equivalent_equals_validator(self):
 class TestEqualsFactory(unittest.TestCase):
 
   def test_with_numbers(self):
-    for expected in [1, 1.0, decimal.Decimal(1), 1L]:
+    for expected in [1, 1.0, decimal.Decimal(1), 1]:
```

Comment:
This last 1 should be `int(1)` with `int` coming from `builtins`, that way it gets a long.
http://python-future.org/compatible_idioms.html#long-integers